src/models/game.py

```diff
@@ -0,0 +1,7 @@
+class Game:
+    """Game holds title, developer, and rating"""
```

Comment:
are we going to store the Genres / Tags with the game? or elsewhere?

Reply:
good question! we'll discuss during sprint planning